### DISCUSSION TITLE
[gatsby-image] How to disable x2 x1.5 width multiplication for mobile, retina display?

### DISCUSSION THREAD

**Discussion by vsolanogo** [gatsbyjs/gatsby#28374] [Help]
I want to it behave on all devices like it is on my development pc. Now if i declare maxWidth: 200 on fluid query on desktop maxWidth will be image with maximum width 200px, but on mobile it will be image with 400px. How to make it 200 everywhere?

**Comment by ascorbic** [comment]
You can set `srcSetBreakpoints` to manually specify your sizes.

**Reply by vsolanogo** [reply]
> 
> 
> You can set `srcSetBreakpoints` to manually specify your sizes.

it doesn't work. Mobile downloads much larger versions, more than 1000px width while i specified [300,400,500,600] src breakdowns. my packages are up to date
